db/repair_bucket.go

```diff
@@ -6,19 +6,23 @@ import (
 
 	"github.com/couchbase/go-couchbase"
 	"github.com/couchbase/sync_gateway/base"
+	"time"
 )
 
 // Enum for the different repair jobs (eg, repairing rev tree cycles)
 type RepairJobType string
 
+const kDefaultRepairedFileTTL = 60 * 60 * 24 * time.Second // 24 hours
+
 const (
 	RepairRevTreeCycles = RepairJobType("RepairRevTreeCycles")
 )
 
 // Params suitable for external (eg, HTTP) invocations to describe a RepairBucket operation
 type RepairBucketParams struct {
 	DryRun            bool              `json:"dry_run"`
-	ViewQueryPageSize int               `json:"view_query_page_size"`
+	ViewQueryPageSize *int              `json:"view_query_page_size"`
+	RepairedFileTTL   *int              `json:"repaired_file_ttl_seconds"`
```

Comment:
Can we rename this to "repair_output_ttl_seconds"?  As written, it sounds like the repaired file itself gets a ttl, which is a bit confusing.

Reply:
Modified parameter name is pushed up